site/docs/getting-started/hello-world.md

```diff
@@ -41,6 +41,17 @@ func update () {
 }
 ```
 
+```odin
+package main
+
+import "cart/w4"
```

Comment:
Is this actual path? Odin's paths are relative by default unless a library collection prefix is given e.g. `import "core:fmt"`

Reply:
Yeah. Lazy C&P is to blame.